src/Pages/ErrorPage/style.module.scss

```diff
@@ -0,0 +1,167 @@
+@import "../../colors.module.css";
+
+/* styling for mobile */
+body{
+  margin: 0px;
+  padding: 0rem !important;
+}
+
+.body{
+  padding: 5vh 0px;
+}
+
+.rightBody{
+  display: flex;
+}
+
+.buttonContainer{
+  display: flex;
+  justify-content: center;
+}
+
+h1, p {
+  color: var(--NAVY_BLUE);
+  text-align: center;
+}
+
+h1 {
+  font: "open sans";
+  font-size: 8rem;
+}
+
+p{
+  font-size: 1.5rem;
+}
+
+.goBackButton{
+  width: 21vw;
+  height: 7vw;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  background-color: var(--BANANA_YELLOW);
+  color: var(--NAVY_BLUE);
+  font-family: 'Open Sans', sans-serif;
+  font-weight: bold;
+  font-size: 1.2rem;
+  border-radius: 8px;
+  border-color: var(--BANANA_YELLOW);
+  border-width: 0px;
+  margin: 10px 10px 10px 1vw;
+}
+
+.homeButton{
+  width: 21vw;
+  height: 7vw;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  background-color: var(--WHITE);
+  color: var(--NAVY_BLUE);
+  font-family: 'Open Sans', sans-serif;
+  font-weight: bold;
+  font-size: 1.2rem;
+  border-radius: 8px;
+  border-color: var(--NAVY_BLUE);
+  border-width: 2px;
+  margin: 10px;
+}
+
+.bananaPeel {
+  position: relative !important;
+  left: -60px !important;
+  top: 349px !important;
+}
+
+.bottomBar{
+  height: 15vh;
+  background-color: var(--BANANA_YELLOW);
+}
+
+.rightBody{
+  justify-content: center;
+  margin-top: 5vh;
+}
+
+/* styling for PC */
+@media (min-width: 900px) {
+  body{
+    margin: 0px;
+    padding: 0rem !important;
+  }
+  
+  .body{
+    padding: 5vh 0px;
+    display: flex;
+    justify-content: center;
+    align-items: start;
+  }
+  
+  .rightBody{
+    display: flex;
+  }
+  
+  .buttonContainer{
+    display: flex;
+    justify-content: center;
+  }
+  
+  h1, p {
+    color: var(--NAVY_BLUE);
+    text-align: center;
+  }
+  
+  h1 {
+    font: "open sans";
+    font-size: 5rem;
+  }
+  
+  p{
+    font-size: 1.5rem;
+  }
+  
+  .goBackButton{
+    width: 21vw;
+    height: 7vw;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    background-color: var(--BANANA_YELLOW);
+    color: var(--NAVY_BLUE);
+    font-family: 'Open Sans', sans-serif;
+    font-weight: bold;
+    font-size: 1.5rem;
+    border-radius: 8px;
+    border-color: var(--BANANA_YELLOW);
+    border-width: 0px;
+    margin: 10px 10px 10px 1vw;
+  }
+  
+  .homeButton{
+    width: 21vw;
+    height: 7vw;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    background-color: var(--WHITE);
+    color: var(--NAVY_BLUE);
+    font-family: 'Open Sans', sans-serif;
+    font-weight: bold;
+    font-size: 1.5rem;
+    border-radius: 8px;
+    border-color: var(--NAVY_BLUE);
+    border-width: 2px;
+    margin: 10px;
+  }
+
+  .bananaPeel {
+    position: relative !important;
```

Comment:
@brendobrendo 
sorry, a little nit-picky.. 
just FYI it's best practice to not use `!important` unless absolutely necessary.

instead, it's better to just get more specific with your CSS selector 
e.g. 
```
.main {
  .404-page {
    .bananaPeel {
      //styles
    }
...

OR 

.main .404page .banana-peel {
  // styles
```
if you can/want-to update thats great, if not this is fine
(this comment applies to all of your `!important` keywords in this PR)

Reply:
Agreed. I was able to change it for the banana image positioning, but not for the body element. Do you know how to create a specific enough selector? I couldn't figure it out and want to avoid using !important when possible